codice/ddf
# DDF-3915 Updated Windows Solr CMD file to avoid logger stack traces

#### What does this PR do?
Replace Solr provided cmd file with one that addresses the stack-traces that appear on boot. 

#### Who is reviewing it? 
@pklinef @oconnormi @rzwiefel @ahoffer @ethantmanns @kcover @bakejeyner @tyler30clemens @figliold 

#### How should this be tested?
Install and run DDF, ensure that there aren't any log4j stack traces in the karaf console.
